fastlane/jetpack_metadata/ja/subtitle.txt

```diff
@@ -1 +0,0 @@
-より速く安全な WordPress
```

Comment:
⚠️  Remember to check that one before final release too (just like you said for `es-ES` above)